docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
Unifying image and root filesystem threshlods might not very useful if the threshold is hardcoded number, instead of a configurable percentage (%). Normally you have a big partition for your image, and small partition for your root. Also normally image size is relatively big, evicting image might resolve out-of-disk.